processing/src/main/java/io/druid/segment/virtual/ExpressionSelectors.java

```diff
@@ -105,7 +105,7 @@ public static DoubleColumnSelector makeDoubleColumnSelector(
     class ExpressionDoubleColumnSelector implements DoubleColumnSelector
     {
       @Override
-      public double get()
+      public double getDouble()
       {
         final Double number = baseSelector.get().asDouble();
```

Comment:
I believe this is out of scope of the PR but I think this poses a potential `NullPointerException`.